src/services/constants.service.ts

```diff
@@ -23,6 +23,7 @@ export class ConstantsService {
     static readonly protectedPin: string = 'protectedPin';
     static readonly clearClipboardKey: string = 'clearClipboardKey';
     static readonly eventCollectionKey: string = 'eventCollection';
+    static readonly biometricUnlockKey = 'fingerprintUnlock';
```

Comment:
We can call the string biometric here. It was left as fingerprint in mobile for backwards compat